test/integration/targets/docker_host_facts/tasks/test_host_facts.yml

```diff
@@ -0,0 +1,249 @@
+---
+- block:
+  - name: Get info on Docker host
+    docker_host_facts:
+    register: output
+
+  - name: assert reading docker host facts when docker is running
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list is not defined'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage is not defined'
+
+  - name: Create random container name prefix
+    set_fact:
+      cname_prefix: "{{ 'ansible-test-%0x' % ((2**32) | random) }}"
+      cnames: []
+      dnetworks: []
+
+  - debug:
+      msg: "Using container name prefix {{ cname_prefix }}"
+
+  - name: Registering container name
+    set_fact:
+      cname: "{{ cname_prefix ~ '-hi' }}"
+  - name: Registering container name
+    set_fact:
+      cnames: "{{ cnames }} + [cname]"
+
+  - name: Create container
+    docker_container:
+      image: alpine:3.8
+      command: '/bin/sh -c "sleep 10m"'
+      name: "{{ cname }}"
+      state: started
+      volumes:
```

Comment:
If you don't need the mounted `/tmp` for this test, I would leave it away. Reduces the probability that this fails because of some bug in `docker_container` (and `docker_container` already has tests for this).